elvis/serializers/serializers.py

```diff
@@ -127,10 +130,13 @@ def _permission_appender(self, result, instance):
             perms = {'can_edit': True, 'can_view': True}
         elif not instance.__dict__.get('public', True):
             perms = {'can_edit': False, 'can_view': False}
+        elif (instance.__class__.__name__ == "Piece" or instance.__class__.__name__ == "Movement") and instance.__dict__.get('hidden', True):
```

Comment:
I believe `elif instance.__dict__.get('hidden', False)` would be equivalent to this line.